Dockerfile

```diff
@@ -57,6 +57,10 @@ RUN echo "Cleaning-up"
 RUN apt-get autoremove && \
     rm -rf /var/lib/apt/lists/*
 
+RUN add-apt-repository ppa:git-core/ppa
+RUN apt-get update
+RUN apt-get install -y git
```

Comment:
For convenience? What's the use case?

Reply:
This is mainly for convenience for development. Perhaps removing the `apt-get install` would be far enough, the docker instance does not have git and `apt` does not know where it is located by default.

Reply:
This was removed because it was not needed.